Formula/gromacs.rb

```diff
@@ -50,7 +50,8 @@ def install
     inreplace "src/gromacs/gromacs-config.cmake.cmakein", "@GROMACS_CXX_COMPILER@", cxx
 
     mkdir "build" do
-      system "cmake", "..", *std_cmake_args, "-DGROMACS_CXX_COMPILER=#{cxx}"
+      system "cmake", "..", *std_cmake_args, "-DGROMACS_CXX_COMPILER=#{cxx}",
+                      "-DGMX_VERSION_STRING_OF_FORK=homebrew"
```

Comment:
```suggestion
      system "cmake", "..", *std_cmake_args, "-DGROMACS_CXX_COMPILER=#{cxx}",
                                             "-DGMX_VERSION_STRING_OF_FORK=#{tap.user}"
```
Let's align the `-D` flags for `cmake`, and use `tap.user` to make this formula easy to reuse for external taps.

Reply:
Done!